src/redis-benchmark.c

```diff
@@ -1771,6 +1823,162 @@ int main(int argc, char **argv) {
 
     tag = "";
 
+#if BENCH_CRC_AVAILABLE
+
+#define RPOLY UINT64_C(0x95ac9329ac4bc9b5)
+#define INIT_SIZE UINT64_C(0xffffffffffffffff)
+
+    if (config.crc64_test_size > 0 || config.crc64_combine) do {
+        unsigned char* data = NULL;
+        if (config.crc64_test_size > 0) {
+            data = zmalloc(config.crc64_test_size);
+            if (!data) {
+                fprintf(stderr, "Could not allocate sufficient data to test crc64 "
+                                "%lld\n", config.crc64_test_size);
+                exit(1);
+            }
+        }
+
+/* Will fix later; minimizing CPU used for getting nstime, as shifts and even
```

Comment:
I don't think we need to fix this. I don't think we need to be that precise about nstime, these benchmarks don't need to be that precise in terms of time taken, just that they are consistent to use as an evaluation tool.